lab.R

```diff
@@ -171,25 +221,28 @@ problem3.4 = 937
 #' 
 #' 1. *For the first problem (column types), we could use the tidyverse function `select()` to pull out a given set of columns from the dataframe.*
 
-# select(dataf, saleprice, overall.cond, gr.liv.area)
+select(dataf, saleprice, overall.cond, gr.liv.area)
 
 #' *But manually typing out all of the numerical covariates would be tedious and prone to error.  Fortunately `select()` is much more powerful than this.  You can read more in `?select` or here: <https://tidyselect.r-lib.org/reference/language.html>.  Then specifically read the docs for `where()`.*
 #' 
 #' *Write a pipe that `select()`s the numeric columns and passes the result to `cor()` for a Spearman regression and uses the `pairwise.complete.obs` method to handle missing values.  Assign the result to `cor_matrix`.*  
-# cor_matrix = ???
+
+cor_matrix = dataf |> 
+  select(where(is.integer)) |> 
+  cor(use="pairwise.complete.obs")
```

Comment:
You want the numeric columns, not just the integers.  And check the docs for `cor()` on how to use Spearman instead of Pearson correlation

Reply:
Got it, thank you! I changed it as requested and pushed to main.